dipy/denoise/localpca.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+import scipy as sp
+import scipy.linalg as sla
+from warnings import warn
+
+# Try to get the SVD through direct API to lapack:
+try:
+    from scipy.linalg.lapack import sgesvd as svd
```

Comment:
actually it's useless to use float64 here, fortran only works in float32 (or at least returns that) but the older version does not (it might actually convert it without telling you, since they only bypass wrapper stuff)

~~~python
from scipy.linalg.lapack import sgesvd as svd
a=np.ones((10,100),np.float64)
u,s,v,info = svd(a)
print(u.dtype)
print(s.dtype)
print(v.dtype)

# older version
from scipy.linalg import svd
u,s,v = svd(a)
print(u.dtype)
print(s.dtype)
print(v.dtype)

## -- End pasted text --
float32
float32
float32
float64
float64
float64
~~~